networkx/algorithms/traversal/depth_first_search.py

```diff
@@ -128,29 +129,28 @@ def dfs_predecessors(G, source=None):
     -----
     Based on http://www.ics.uci.edu/~eppstein/PADS/DFS.py
     by D. Eppstein, July 2004.
-
+    https://en.wikipedia.org/wiki/Depth-limited_search
     If a source is not specified then a source is chosen arbitrarily and
     repeatedly until all components in the graph are searched.
     """
-    return dict((t,s) for s,t in dfs_edges(G,source=source))
+    return dict((t, s) for s, t in dfs_edges(G, source=source,
+    depth_limit=depth_limit))
```

Comment:
line continuations should be indented... usually up to the open parentheses that makes them a line continuation--but whatever makes it look good. If you have more than one line continuation the should be indented the same amount.